pilot/pkg/proxy/envoy/v1/watcher.go

```diff
@@ -85,6 +85,8 @@ func (w *watcher) Run(ctx context.Context) {
 	// agent consumes notifications from the controller
 	go w.agent.Run(ctx)
 
+	time.Sleep(time.Second * 30)
```

Comment:
this needs to be a wait for the cert not a sleep

Reply:
that's just experimental :) I fixed it.